django/utils/feedgenerator.py

```diff
@@ -166,6 +168,12 @@ def add_root_elements(self, handler):
         """
         pass
 
+    def add_stylesheets(self, handler):
+        """
+        Add stylesheet(s) to the feed (XSLT).
+        """
```

Comment:
The corresponding doc you've added in `docs/ref/utils.txt` says: 
> [...] Called from ``write()``

I would assume then, that like the methods just above this, the docstring itself ought to say the same for consistency? Maybe? 🤷